src/index.ts

```diff
@@ -361,7 +356,9 @@ prog
       argv.push('--watchAll');
     }
 
-    const maybeTestSetupFiledExists = await fs.exists(paths.testsSetup);
+    const maybeTestSetupFiledExists = await util.promisify(fs.exists)(
```

Comment:
Forgot to wrap `fs.exists` with `util.promisify`

Reply:
fs-extra is 100% promise

Reply:
The `@types/fs-extra` package seems to forward many function types through without adding promise overloads. lol

Reply:
That's odd.

Reply:
I take that back. It was just an inference issue. But the `overwrite` option is missing from the typedefs for sure.

Reply:
overwrite isn't an option. I think i must have copypasted from fs.copy. you can remove

Reply:
I assume `fs.exists` is missing that promise overload because it's technically deprecated